ipi/inputs/motion/neb.py

```diff
@@ -150,13 +183,61 @@ class InputNEB(InputDictionary):
                 "help": "The number of past vectors to store for L-BFGS.",
             },
         ),
+        "dtmax_fire": (
+            InputValue,
+            {
+                "dtype": float,
+                "default": 1.0,
+                "help": "Maximum time interval per step for FIRE.",
+            },
+        ),
+        "v_fire": (
+            InputArray,
+            {
+                "dtype": float,
+                "default": input_default(factory=np.zeros, args=(0,)),
+                "help": "Current velocity for FIRE",
+            },
+        ),
+        "alpha_fire": (
+            InputValue,
+            {
+                "dtype": float,
+                "default": 0.1,
+                "help": "velocity mixing factor for FIRE",
+            },
+        ),
+        "N_down_fire": (
+            InputValue,
+            {
+                "dtype": int,
+                "default": 0,
+                "help": "consecutive steps in downhill dierction for FIRE",
+            },
+        ),
+        "N_up_fire": (
+            InputValue,
+            {
+                "dtype": int,
+                "default": 0,
+                "help": "consecutive steps in uphill direction",
+            },
+        ),
+        "dt_fire": (
+            InputValue,
+            {
+                "dtype": float,
+                "default": 0.1,
+                "help": "time per step",
+            },
+        ),
         "endpoints": (
             InputDictionary,
             {
                 "dtype": [bool, str],
                 "options": ["optimize", "algorithm"],
-                "default": [True, "bfgs"],
-                "help": "Geometry optimization of endpoints",
+                "default": [False, "bfgs"],
+                "help": "Geometry optimization of endpoints (not implemented yet)",
```

Comment:
is there a safe exit in case someone puts true here?

Reply:
Yes, it's safe